igneous/tasks.py

```diff
@@ -264,7 +265,14 @@ def __init__(self, shape, offset, layer_path, **kwargs):
         'low_padding': kwargs.get('low_padding', 0),
         'high_padding': kwargs.get('high_padding', 1),
         'parallel_download': kwargs.get('parallel_download', 1),
-        'cache_control': kwargs.get('cache_control', None)
+        'cache_control': kwargs.get('cache_control', None),
+        'use_draco_encoding': kwargs.get('use_draco_encoding', False),
```

Comment:
It might be more backwards compatible to write `'encoding': 'draco'` in case we ever add more options.

Reply:
Agreed